### ISSUE TITLE
[🐛 Bug]: Missing pom file for org.seleniumhq.selenium:selenium-devtools-v119:4.16.0

### ISSUE THREAD

**Issue by dietzsch** [SeleniumHQ/selenium#13256]
### What happened?

We have seen that today a new version of Selenium 4.16.0 was released. When updating the Selenium version to 4.16.0 the build fails as the transitive dependency on org.seleniumhq.selenium:selenium-devtools-v119:4.16.0 can not be resolved. From what we can see on MavenCentral the jar file, javadoc are present, but the pom file is missing even when the checksum/signature for that file is available.

### How can we reproduce the issue?

```shell
we have a Java project based on Gradle with a dependency like this:

    testImplementation "org.seleniumhq.selenium:selenium-java:4.16.0"
```

### Relevant log output

```shell
* What went wrong:
Execution failed for task ':vmaas-password-encoder:compileTestGroovy'.
> Could not resolve all files for configuration ':vmaas-password-encoder:testCompileClasspath'.
   > Could not resolve org.seleniumhq.selenium:selenium-devtools-v119:4.16.0.
     Required by:
         project :vmaas-password-encoder > org.seleniumhq.selenium:selenium-java:4.16.0
      > Could not resolve org.seleniumhq.selenium:selenium-devtools-v119:4.16.0.
         > Could not get resource 'https://repo.spring.io/plugins-release/org/seleniumhq/selenium/selenium-devtools-v119/4.16.0/selenium-devtools-v119-4.16.0.pom'.
            > Could not GET 'https://repo.spring.io/plugins-release/org/seleniumhq/selenium/selenium-devtools-v119/4.16.0/selenium-devtools-v119-4.16.0.pom'. Received status code 401 from server: 
      > Could not resolve org.seleniumhq.selenium:selenium-devtools-v119:4.16.0.
         > Could not get resource 'https://repo.spring.io/release/org/seleniumhq/selenium/selenium-devtools-v119/4.16.0/selenium-devtools-v119-4.16.0.pom'.
            > Could not GET 'https://repo.spring.io/release/org/seleniumhq/selenium/selenium-devtools-v119/4.16.0/selenium-devtools-v119-4.16.0.pom'. Received status code 401 from server:
```

### Operating System

CentOS

### Selenium version

Selenium Java 4.16.0

### What are the browser(s) and version(s) where you see this issue?

n/a

### What are the browser driver(s) and version(s) where you see this issue?

n/a

**Comment by asolntsev** [comment]
I am using a simple workaround - just exclude this dependency in my `build.gradle`:

```groovy
dependencies {
    ...

    testImplementation("org.seleniumhq.selenium:selenium-java:4.16.0") {
      exclude group: 'org.seleniumhq.selenium', module: 'selenium-devtools-v119' // issue https://github.com/SeleniumHQ/selenium/issues/13256
    }

    ...
}
```

**Comment by titusfortner** [comment]
Fixed in 4.16.1
